ntbs-service/Pages/Notifications/OverviewPartials/_TestResultsOverviewPartial.cshtml

```diff
@@ -94,7 +94,7 @@
 
     <nhs-grid-row>
         <nhs-grid-column grid-column-width="OneQuarter">
-            <div class="notification-details-label"> @Html.DisplayNameFor(m => m.Notification.TestData.HasTestCarriedOut) </div>
+            <div class="notification-details-label"> @Html.DisplayNameFor(m => m.Notification.TestData.HasTestCarriedOut)? </div>
```

Comment:
I removed it from the attribute display name, so had to add it here.
Reason I removed it was because it was creating stupid-looking error messages like "The field Have any tests been carried out? must be empty" (not exactly this, but you get the gist ;) )